File: backend/src/main/java/com/zzang/chongdae/comment/service/dto/LatestCommentResponse.java
package com.zzang.chongdae.comment.service.dto;

import com.zzang.chongdae.comment.repository.entity.CommentEntity;
import java.time.LocalDateTime;

public record LatestCommentResponse(String content, LocalDateTime createdAt) {

    public LatestCommentResponse(CommentEntity comment) {
        this(comment.getContent(), comment.getCreatedAt());
    }
}

File: android/app/src/main/java/com/zzang/chongdae/data/remote/dto/response/LatestCommentResponse.kt
package com.zzang.chongdae.data.remote.dto.response

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class LatestCommentResponse(
    @SerialName("content") val content: String,
    @SerialName("createdAt") val createdAt: String,
)
